virttest/qemu_vm.py

```diff
@@ -1271,6 +1278,8 @@ def add_balloon(devices, devid=None, bus=None, use_old_format=None):
             dev = QDevice(model, parent_bus=bus)
             if devid:
                 dev.set_param("id", devid)
+            for key, value in six.iteritems(kwargs):
```

Comment:
I suggest use `for key, value in kwargs.items():` directly.
The rest are ok to me.

Reply:
It is different, but it does not matter in this case.
Because you didn't use the functionality provided by `iteritems()` here, and yet you used `six` instead of python build-ins.
So  you don't need to do any changes, if you use `kwargs.items()` which is python build-in.